### ISSUE TITLE
Notes should support html links

### ISSUE THREAD

**Issue by SimonWahlin** [PalmEmanuel/FoldTheDishes#36]
I want to paste html links from an email or a website to the notes field and have them "clickable" later.

**Comment by PalmEmanuel** [comment]
Good idea! I can't figure out a reasonable way to implement it with the current edit view since the field is simply editable text, so it will require implementing a separate ReadOnly view of the ReminderDetail page. Should be possible though, definitely worth taking a look at.

**Comment by SimonWahlin** [comment]
I can paste the links in the edit view and click them, but once the task is saved it only keeps plain text.
